chapter04/Task_4_4.py

```diff
@@ -0,0 +1,29 @@
+# Guess the word
+#
+# The computer makes a word and answers only "Yes" or "no".
+#
+# The player has 5 attempts to guess the maximum number of letters, and then the player must write the entire word
+
+import random
+
+WORDS = ("python", "jumble", "easy", "difficult", "answer", "xylophone")
+word = random.choice(WORDS)
+
+print('In guess word', len(word), 'letters\n')
+
+i = 0
+
+while i < 5:
```

Comment:
This case looks perfect for the "for" loop.
In general, if case is naturally suitable for "for" (without any hacks), it's better to use this one as it's easier to read because you have expectations about how that behaves and you shouldn't track waht is updated and what is the rule.

Reply:
I disagree. Need clarification. Do you really want me to create an endless loop and then exit it?

Is such a decision acceptable?